core/src/main/java/org/axonframework/eventhandling/saga/ThrowingSagaErrorHandler.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2010-2017. Axon Framework
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ *
+ */
+
+package org.axonframework.eventhandling.saga;
+
+import org.axonframework.eventhandling.EventMessage;
+
+/**
+ * Implementation that simply throws the reported exception.
+ *
+ * @author Milan Savic
+ * @since 3.2
+ */
+public enum ThrowingSagaErrorHandler implements SagaInvocationErrorHandler {
```

Comment:
I see you created this class, following the example of the classes already present. I don't think this instance is necessary. The existing `ThrowingListenerErrorHandler` can be marked `@Deprecated` to be removed in future versions. They don't really add any value.